### PR TITLE
Bug 1513552 add modules ping parquet schema

### PR DESCRIPTION
**Pull request by CarlCorcoran**
This PR adds a parquet schema for the "Modules" ping in order to make it available on redash.

- Part 1 puts the common ping environment schema into an include file.
- Part 2 adds the actual modules parquet schema files.

See commit messages for further details

Checklist for reviewer:

- [ ] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [ ] Trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional)

### CODE REVIEW

**Comment by CarlCorcoran** [comment]
Is the existing modules validation ping sufficient? My changes should just be a parquet translation of the existing modules ping JSON schema. See the existing validation ping here: https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/dev/validation/telemetry/modules.4.windows.pass.json

The untrusted modules ping also has a existing validation pings.

File: templates/telemetry/modules/modules.4.parquetmr.txt [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  optional binary clientId (UTF8);
+  required binary creationDate (UTF8);
+  required binary id (UTF8);
+  required binary type (UTF8);
+  required double version;
+  required group payload {
+    required int64 version;
+    optional group modules (LIST) {
+      repeated group list {
+        optional group element (TUPLE) {
```
Comment (fbertsch) [inline templates/telemetry/modules/modules.4.parquetmr.txt]: Should this be a `TUPLE`? See here for an explanation of lists: https://github.com/apache/parquet-format/blob/master/LogicalTypes.md

It seems to me this should just be a normal group with 4 fields.

Comment (CarlCorcoran) [reply templates/telemetry/modules/modules.4.parquetmr.txt]: Done; fixed!

Comment (trink) [reply templates/telemetry/modules/modules.4.parquetmr.txt]: fyi: TUPLE is our extension and is not documented at the link above see: https://bugzilla.mozilla.org/show_bug.cgi?id=1355553

**Comment by CarlCorcoran** [comment]
Thanks @fbertsch ! Do you know what steps are needed from here before we can see this on STMO?

**Comment by mreid-moz** [comment]
cc @whd for a deploy

**Pull request merged**